auth/users.go

```diff
@@ -30,3 +38,49 @@ func (k *KratosHandler) createRecoveryLink(id string) (string, error) {
 	}
 	return resp.GetRecoveryLink(), nil
 }
+
+func (k *KratosHandler) createAdminIdentity() (string, error) {
+	var adminPassword string
+	adminPassword = os.Getenv("ADMIN_PASSWORD")
```

Comment:
I think we need to wire this through to the helm chart

Reply:
@moshloop Have created the PR for that here: https://github.com/flanksource/incident-commander-chart/pull/15